src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -2274,9 +2274,19 @@ static void updateAttrInNotifyCer
       //
       // FIXME P6: https://github.com/telefonicaid/fiware-orion/issues/2587
       // If an attribute has no value, then its value is not updated (neither is previousValue).
-      // However this may be problematic ... see the issue
+      // However this may be problematic ... see the issue.
       //
+      // New data on this: the functest "null_not_working_in_q_for_subscription.test" fails with this 'if-clause' (not outdeffed),
```

Comment:
case/XXXX/null_not_working_in_q_for_subscription.test
case/XXXX/only q_and_mq_as_uri_param_for_metadata.test

will help to locate these .test files in the future

(I mean just casesXXX/, I think case/XXXX_long_name_sometimes_very_long would make thinks worse)

Reply:
I assumed people would be bright enough to issue a
`find . -name null_not_working_in_q_for_subscription.test`.
You think not? :-)

Sure, I can add indications on how to find the file

Reply:
Fixed in ba413e8

Reply:
Of course user can use grep or any other similar tool in its operations system but... why don't provide extra help in the comment itself? ;)